app/src/main/java/com/framgia/englishconversation/screen/createcomment/CreateCommentViewModel.java

```diff
@@ -218,7 +216,7 @@ public void onSaveEditComment(View view) {
         mComment.setModifiedAt(System.currentTimeMillis());
         mComment.setContent(mInputtedComment);
         MediaModel mediaModel = mComment.getMediaModel();
-        if (mediaModel == null || (mediaModel.getUrl()).toLowerCase().startsWith(LINK_REMOTE)) {
+        if (mediaModel == null || (mediaModel.getUrl()).toLowerCase().startsWith(HTTPS)) {
```

Comment:
WARNING: [DefaultLocale] Implicitly using the default locale is a common source of bugs: Use `toLowerCase(Locale)` instead. For strings meant to be internal use `Locale.ROOT`, otherwise `Locale.getDefault()`.
 Calling `String#toLowerCase()` or `#toUpperCase()` **without specifying an explicit locale** is a common source of bugs. The reason for that is that those methods will use the current locale on the user's device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is **not** `I`.

If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead.